casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -309,7 +309,7 @@ class RuntimeManagerImpl[F[_]: Concurrent: Metrics: Span: Log](
           (startHash, processedDeploys) = deployProcessResult
           systemDeployProcessResult <- {
             import cats.instances.list._
-            systemDeploys.toList.foldM((startHash, Vector.empty[ProcessedSystemDeploy])) {
+            systemDeploys.toVector.foldM((startHash, Vector.empty[ProcessedSystemDeploy])) {
```

Comment:
`systemDeploys` will be a List here so better to not change it. Although this list should not be big enough to make a difference.

P.S.
With latest cats update it's no longer needed to import instance when `traverse` (`foldM`) is used. In cats v3 (or 2.3) Seq will have instance for Traverse so explicit conversion should also be redundant.
https://github.com/typelevel/cats/pull/3620

https://github.com/typelevel/cats/pull/3620

Reply:
Changed!